src/ApplicationInsights.Kubernetes.HostingStartup/K8sInjection.cs

```diff
@@ -3,8 +3,15 @@
 
 namespace ApplicationInsights.Kubernetes.HostingStartup
 {
+    /// <summary>
+    /// Hosted Startup point for Application Insights for Kubernetes.
```

Comment:
Call it Hosting Startup instead of Hosted Startup to match the name IHostingStartup